packages/std/src/math/uint64.rs

```diff
@@ -204,6 +204,10 @@ impl Uint64 {
     pub fn saturating_pow(self, other: u32) -> Self {
         Self(self.0.saturating_pow(other))
     }
+
+    pub const fn abs_diff(self, other: Self) -> Self {
+        Self(self.0.abs_diff(other.0))
```

Comment:
Same as the u128 case:

```suggestion
        Self(if self.0 < other.0 {
            other.0 - self.0
        } else {
            self.0 - other.0
        })
```